apps/system/js/core.js

```diff
@@ -110,11 +109,12 @@
         'MultiScreenController',
         'PinsManager',
         //IFDEF_FIREFOX_SYNC
-        'SyncManager'
+        'SyncManager',
         //ENDIF_FIREFOX_SYNC
         // XXX: We should move CarrierInfoNotifier into mobileConnectionCore,
         // but integration tests running on desktop without mobileConnection
         // is testing this.
+        'CustomModules'
```

Comment:
That was the same it was doing, wasn't it? https://github.com/mozilla-b2g/gaia/pull/34118/files#diff-135794ad5f1433b712b86b905dbfab95L70